Sources/ProjectDescription/Target.swift

```diff
@@ -18,6 +18,12 @@ public class Target: Codable {
     /// Relative path to the Info.plist file.
     public let infoPlist: String
 
+    /// Name of the storyboard to be used for the main storyboard
+    public let mainStoryboard: String?
```

Comment:
When users edit these models using Xcode they have access to the documentation. Considering that both added attributes require the name to not have the extension, I'd document it:

````
Name (without extension) of the storyboard to be used for the main storyboard.
```